vflow/convert.py

```diff
@@ -84,6 +86,38 @@ def dict_to_df(d: dict, param_key=None):
                 df = df.iloc[:, new_idx]
     return df
 
+def evaluate_uncertainty(mean_preds, std_preds):
```

Comment:
Let's make this work with more general output from `Vset.predict_with_uncertainties`, i.e. when `group_by` is used, by returning `uncertainty` and `cum_acc` as a list with one entry for each matched entry in `mean_preds` + `std_preds`. Then we can, for example, plot two models separately:
```
u, c = evaluate_uncertainty(mean_dict, std_dict)
plt.plot(u[0], c[0]) # DT
plt.plot(u[1], c[1]) # LR
```